notebooks/interfield_differences.ipynb

```diff
@@ -0,0 +1,1245 @@
+{
```

Comment:
<p>Interesting that three of the top 5 (cb - ai) papers are about COVID - I wonder if that trend holds across more highly ranked papers?</p><p> </p><p>Also interesting that the low (cb - ai) papers aren't really explicitly "methods-y", I would have expected them to sound more AI-oriented from the titles.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Yeah, I think that shuffling the networks individually caused more noise; the joint-shuffled versions seemed more reasonable</p>